cdci_data_analysis/analysis/parameters.py

```diff
@@ -710,10 +695,10 @@ def check_energy_value(value, units=None,name=None):
             raise RuntimeError('type of ',name,'not valid',type(value))
 
 
-class Energy(Parameter):
+class Energy(Float):
```

Comment:
this is (almost? Or completely?) identical to `SpectralBoundary`

Reply:
the only difference is the `_allowed_units` list, perhaps that's what really differentiate them

Reply:
also, Energy does not have a `units` set, should it have one?

Reply:
it should!

Reply:
ok, which one should it be?

Reply:
allowed units should be the same. default units should be `keV`